### DISCUSSION TITLE
#[Locked] property not working properly

### DISCUSSION THREAD

**Discussion by viliusvsx** [livewire/livewire#7052] [7. Bugs (archived)] [answered]
### Livewire version

v3.0.8

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Create a new project with the Livewire component.
2. Create 3 attributes like `id`, `title` and `body`. In view file add 2 inputs with `wire:model.live` for `title` and `body`
3. Set #[Locked] attribute to `id`.
4. Try to change the `title` or `body` in the blade file.

At this point you should get an error that `Cannot update locked property: [id]`, although I didn't edit that. 

P.S. In Livewire v3.0.5 this error did not occur

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by swissinside** [comment, marked as answer]
A fix is already available as PR: #7060
